terraform/modules/front_door/waf.tf

```diff
@@ -285,7 +285,6 @@ resource "aws_wafv2_regex_pattern_set" "waf_rate_limit_urls" {
 
 resource "aws_wafv2_ip_set" "allowed_ips" {
   provider = aws.us-east-1
-  count    = var.restrict_by_ip ? 1 : 0

```

Comment:
Why would we no longer want this?

Reply:
In this case, mainly because deleting it when turning the restriction off is annoying (mainly for ordering reasons - which we could sort out differently but it was easier not to for now). Will come back to this when sorting out which envs the ip restriction is on for and the remaining ips.